Nightwatch.Tests/Nightwatch.Tests.fsproj

```diff
@@ -19,5 +19,6 @@
     <PackageReference Include="xunit" Version="2.2.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
     <ProjectReference Include="../Nightwatch/Nightwatch.fsproj" />
+    <PackageReference Include="TaskBuilder.fs" Version="1.2.0-rc" />
```

Comment:
I bet test assembly do inherit the dependency from the main assemblies. Is it possible to drop the dependency from this particular fsproj?